### PR TITLE
Updated gce node-termination-handler yaml.

### PR DESCRIPTION
**Pull request by yuwenma**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

/kind cleanup

> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
We push a new release for image node-termination-handler which is based on distroless.
[new image](https://pantheon.corp.google.com/gcr/images/google-containers/GLOBAL/gke-node-termination-handler@sha256:aca12d17b222dfed755e28a44d92721e477915fb73211d0a0f8925a1fa847cca/details?tab=info) 
Image Rebase: GoogleCloudPlatform/k8s-node-termination-handler#12
taint flag: GoogleCloudPlatform/k8s-node-termination-handler#13
example yaml: GoogleCloudPlatform/k8s-node-termination-handler#14

**Which issue(s) this PR fixes**:

Fixes #
GoogleCloudPlatform/k8s-node-termination-handler#13

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cluster/gce/addons/node-termination-handler/daemonset.yaml [hunk 1]
```diff
@@ -48,10 +48,10 @@ spec:
         - effect: NoExecute
           operator: Exists
       containers:
-        - image: k8s.gcr.io/gke-node-termination-handler@sha256:e08ca863a547754fa7b75064bdad04f04cbef86c7b0a181ecc7304e747623181
+        - image: k8s.gcr.io/gke-node-termination-handler@sha256:aca12d17b222dfed755e28a44d92721e477915fb73211d0a0f8925a1fa847cca
           name: node-termination-handler
           command: ["./node-termination-handler"]
-          args: ["--logtostderr", "--exclude-pods=$(POD_NAME):$(POD_NAMESPACE)", "-v=10", "--kubeconfig=/var/lib/kubelet/kubeconfig", "--annotation=cloud.google.com/impending-node-termination"]
+          args: ["--logtostderr", "--exclude-pods=$(POD_NAME):$(POD_NAMESPACE)", "-v=10", "--kubeconfig=/var/lib/kubelet/kubeconfig", "--annotation=cloud.google.com/impending-node-termination", "--taint=cloud.google.com/impending-node-termination::NoSchedule"]
```
Comment (vishh) [inline cluster/gce/addons/node-termination-handler/daemonset.yaml]: @yuwenma Can you remove annotations here? That feature is yet to be rolled out.

**Comment by yuwenma** [comment]
/assign @gmarek
/assign @yujuhong

The auto-assigned approver no longer works in this area. Reassign to other approvers of the gce dir.

**Pull request merged**